src/graphql/node/control.rs

```diff
@@ -39,188 +38,249 @@ impl NodeControlMutation {
         }
     }
 
-    // TODO: Apply node issue #251
+    /// Applies the draft configuration to the node with the given ID.
+    ///
+    /// This function updates the node's `name` with `name_draft`, `profile` with `profile_draft`,
+    /// and `config` values of agents with their `draft` values. The function returns success as
+    /// long as the database update is successful, regardless of the outcome of notifying agents or
```

Comment:
I suggest putting the following:

... `draft` values.

Returns success as long as ...

Reply:
Sorry for the confusion. I should've posted this in the `applyNode` PR.